internal/serve/httphandler/payments_handler.go

```diff
@@ -0,0 +1,33 @@
+package httphandler
+
+import (
+	"net/http"
+	"wallet-backend/internal/data"
+	"wallet-backend/internal/serve/httperror"
+
+	"github.com/stellar/go/support/http/httpdecode"
+)
+
+type PaymentsHandler struct {
+	*data.Models
+}
+
+type PaymentsSubscribeRequest struct {
+	Address string `json:"address"`
+}
+
+func (h PaymentsHandler) SubscribeAddress(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	var reqBody PaymentsSubscribeRequest
+	if err := httpdecode.DecodeJSON(r, &reqBody); err != nil {
+		httperror.BadRequest("Invalid request body").Render(w)
+		return
+	}
```

Comment:
nit: it would be great if we can have a consistent error handling pattern in a single function
```suggestion
	err := httpdecode.DecodeJSON(r, &reqBody)
	if err != nil {
		httperror.BadRequest("Invalid request body").Render(w)
		return
	}
```